airbyte-integrations/connectors/source-marketo-singer/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,150 @@
+            },
+            "workspaceName": {
+              "type": ["string", "null"]
+            }
+          }
+        },
+        "supported_sync_modes": ["incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": []
+      },
+      "sync_mode": "incremental"
+    },
+    {
+      "stream": {
+        "name": "lists",
+        "json_schema": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "id": {
+              "type": "integer"
+            },
+            "name": {
+              "type": "string"
+            },
+            "createdAt": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "updatedAt": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": {
+              "type": ["string", "null"]
+            },
+            "programName": {
+              "type": ["string", "null"]
+            },
+            "workspaceName": {
+              "type": ["string", "null"]
+            }
+          }
+        },
+        "supported_sync_modes": ["incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": []
+      },
+      "sync_mode": "incremental"
+    },
+    {
+      "stream": {
+        "name": "programs",
+        "json_schema": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "id": {
+              "type": "integer"
+            },
+            "createdAt": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "updatedAt": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": ["null", "string"]
+            },
+            "url": {
+              "type": ["null", "string"]
+            },
+            "type": {
+              "type": ["null", "string"]
+            },
+            "channel": {
+              "type": ["null", "string"]
+            },
+            "status": {
+              "type": ["null", "string"]
+            },
+            "workspace": {
+              "type": ["null", "string"]
+            },
+            "folder": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": ["null", "string"]
+                },
+                "value": {
+                  "type": ["null", "integer"]
+                },
+                "folderName": {
+                  "type": ["null", "string"]
+                }
+              }
+            }
+          }
+        },
+        "supported_sync_modes": ["incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": []
+      },
+      "sync_mode": "incremental"
```

Comment:
@yevhenii-ldv can we also add some of the activities streams to this catalog?

Reply:
@sherifnada no, unfortunately this is impossible, because to get data of each of the activites, the code inside `tap-marketo` creates a `Worker`, then starts it, sets sleep for 5 minutes (the constant is indicated in the code) in order to wait until the Worker processes the request, and after 5 minutes it checks ready whether a `Worker`, if it returns the "Completed" `status`, then `tap-marketo` downloads the `csv `file, parses it and outputs the data. And such a `Worker` is created for each month (for example: 01/01/2020 - 01/31/2020, 02/01/2020 - 02/29/2020). Thus, integration tests will take an incredibly long time.